curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-oop-by-building-a-platformer-game/64c9fe7b2ffa3539fbf82d32.md

```diff
@@ -0,0 +1,341 @@
+---
+id: 64c9fe7b2ffa3539fbf82d32
+title: Step 81
+challengeType: 0
+dashedName: step-81
+---
+
+# --description--
+
+Add one last conditional statement that checks if the player's `x` position is less than or equal to the sum of the platform's `x` position plus the platform's width minus one third of the player's width.
+
+# --hints--
+
+TODO: Tried adding tests but none of my attempts were working. Will try again later.
+
+```js
+
+```
```

Comment:
You should have a conditional statement that checks if the player's `x` position is lesser than or equal to the platform's `x` position plus the platform's width minus the player's width divided by `3`.

```js
assert.match(code, /player\.position\.x\s*<=\s*platform\.position\.x\s*\+\s*platform\.width\s*-\s*player\.width\s*\/\s*3,?/)
```